oars_gb_pkg/nodes/sensing/wind_filter.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+import rospy
+import numpy as np
+from collections import deque
+from std_msgs.msg import UInt8
+from geometry_msgs.msg import Pose2D
+
+'''
```

Comment:
Super nitpicky thing: while this works fine, the PEP8 style guide says one should use triple double-quotes (`"""`) for docstrings. (+1 for the docstring though)